### DISCUSSION TITLE
Template Installation Error

### DISCUSSION THREAD

**Discussion by redbullsummer** [bonsai-rx/bonsai#928] [Q&A] [answered]
Hi,

I'm having issues running the template installer with Visual Studio. I am currently on Visual Studio Community 2022 17.2.3 and the installation is failing to install. I have tried modifying by adding more development packages, but nothing seems to work.
![image](https://user-images.githubusercontent.com/107198572/172874168-9f1fd09d-b06a-42df-b639-b21819c0b04b.png)
![image](https://user-images.githubusercontent.com/107198572/172874390-67615a4b-6dc2-4c96-b02b-ac5bbd8d4ffc.png)

***Update
Installed 2019 version and installed perfectly

**Comment by bruno-f-cruz** [comment, marked as answer]
This issue (#832) has been flagged and fixed. It will be officially released with Bonsai 2.7. Stay tuned :)
